bloco_3/dia_3/Ex. 4/style4.css

```
@@ -0,0 +1,74 @@
+/*
+
+1 - Faça com que todos os itens de Listas Ordenadas tenham uma cor de fundo amarela. Se a numeração do item for PAR, faça a cor de fonte ser verde. Se o número for considerado primo, utilize a propriedade text-transform para deixar o texto maiúsculo. (Dica: combinar classes pode ser útil aqui).
+
+2 - Faça todas as tags Header (h1, h2...) possuírem cor de fonte vermelha e, se alguma delas estiver em itálico, aumente seu tamanho para 40px e acrescente uma borda de 1px preta e sólida.
+
+3 - Faça todas as li's terem 20px de tamanho de fonte e, para todo texto em itálico dentro de alguma li, utilize a propriedade font-weight para deixá-lo negrito.
+
+4 - Na listagem de linguagens de programação, faça com que as 3 primeiras da lista possuam cor de fonte azul e, se alguma possuir "java*" no texto, utilize uma cor de fundo rosa.
+
+5 - (Bônus) Para cada link na lista de aliados da pessoa desenvolvedora, faça com que ao passar o mouse sobre cada um, o texto fique em negrito e assuma a cor de fonte "temática do site"
+  Dica 1: Utilize o seletor :hover para controlar o passar do mouse.
+  Dica 2: Sugestão de cores padrão. StackOverflow (laranja), W3Schools (verde), MDN (preta), DevDocs (amarela), Trybe (verde).
+
+*/
+
+.bg-ol {
+  background-color: yellow;
+  font-weight: 700;
+}
+
+.bg-ol li:nth-of-type(even) {
+  background-color: green;
+}
+/* FALTA A POHA DO NUMERO PRIMO */
+.bg-ol li:nth-child(2) {
+  text-transform: uppercase;
+}
+
+.headers {
+  color: red;
+}
+
+h1.headers {
+  font-size: 40px;
+  border: 1px solid black;
+}
+
+.all-li {
+  font-size: 20px;
+}
+
+#python-blue {
+  color: blue;
+}
+
+.blue-java {
+  color: blue;
+  background-color: pink;
+}
+
+.hover-helpers:hover {
+  font-weight: bold;
+}
+
+.hover-helpers:hover li:first-child {
+  color: orange;
+}
+
+.hover-helpers:hover li:nth-child(2) {
+  color: green;
+}
+
+.hover-helpers:hover li:nth-child(3) {
+  color: black;
+}
+
+.hover-helpers:hover li:nth-child(4) {
+  color: yellow;
+}
+
+.hover-helpers:hover li:nth-child(5) {
```

Para poupar linha de codigo voce pode adicionar uma `,` na linha do segundo filho e adicioanr esse quinto filho nela.

```suggestion
.hover-helpers:hover li:nth-child(2), .hover-helpers:hover li:nth-child(5) {
```